test/it/uk/gov/hmrc/component/BaseFeatureSpec.scala

```diff
@@ -66,8 +66,8 @@ abstract class BaseFeatureSpec extends FeatureSpec
 
   protected def storeCases(cases: Case*): Seq[Case] = {
```

Comment:
I did not write any integration tests using the encryption, because `application.conf` has it disabled.
However, we have the acceptance tests in QA use the encryption..